xchange-bitfinex/src/main/java/info/bitrich/xchangestream/bitfinex/BitfinexStreamingExchange.java

```diff
@@ -57,4 +59,28 @@ public StreamingMarketDataService getStreamingMarketDataService() {
     @Override
     public void useCompressedMessages(boolean compressedMessages) { streamingService.useCompressedMessages(compressedMessages); }
 
+    public Completable connectToAuthenticated() {
```

Comment:
Is it possible to make this connection/authentication transparent, so it occurs automatically if authentication details are available, as we do for GDAX?

Reply:
Done on my uplift fork